src/test/suite/fixtures/recorded/languages/rust/changeTypeUrge.yml

```diff
@@ -0,0 +1,32 @@
+languageId: rust
+command:
+  spokenForm: change type urge
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: u}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: type}
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: |
+    const X: [u8; 2] = [0, 0];
+  selections:
+    - anchor: {line: 1, character: 0}
+      active: {line: 1, character: 0}
+  marks:
+    default.u:
+      start: {line: 0, character: 10}
+      end: {line: 0, character: 12}
+finalState:
+  documentContents: |
+    const X: [; 2] = [0, 0];
```

Comment:
Huh I think I'd expect the whole `[u8; 2]` to be `"type"` here? I don't feel super strongly but that would be my leaning. Cc/ @auscompgeek @AndreasArvidsson

Reply:
Agreed. I would expect the entire `[u8, 2]` to be the type

Reply:
Hmm, I think this behaviour is reasonable, not sure how I'd select the type the array contains otherwise if it's more complex, e.g. `[HashMap<u32, u64>; 4]`. The array type can be reasonably selected in other ways.

Reply:
> Hmm, I think this behaviour is reasonable, not sure how I'd select the type the array contains otherwise if it's more complex, e.g. `[HashMap<u32, u64>; 4]`. The array type can be reasonably selected in other ways.

My reasoning exactly. The whole type can be selected with `take type plex` etc.

Reply:
Not at all

Reply:
Cool so let's just leave this one as is for now.  We can add a special scope type for `length` later if we feel we need it 🤷